### DISCUSSION TITLE
Customization of Local builder

### DISCUSSION THREAD

**Discussion by ZaheerNiazi007** [chaibuilder/sdk#125] [General]
Hi! What an amazing builder! I really love it. However, I have a question—can we customize the local builder with built-in blocks? For example, I want the existing blocks to remain as they are, but I also want to add more, like this:
<ChaiBuilderLocal
  // blocks={[
  //   {
  //     _type: "Heading",
  //     _id: "a",
  //     content: "This is a heading",
  //     styles: "#styles:,text-3xl font-bold",
  //   },
  // ]}
  onSave={async ({ blocks, providers, brandingOptions }) => {
    console.log(blocks, providers, brandingOptions);
    return true;
  }}
/>
Is this possible, or can we only edit the SDK?"

**Comment by surajair** [comment]
hi,
`I want the existing blocks to remain as they are, but I also want to add more, like this`
Do you want to register your own blocks similar to core blocks like heading, parapgraphs etc?

**Reply by ZaheerNiazi007** [reply]
@surajair Yes, I have already implemented it like this.
Thank for reply

**Reply by surajair** [reply]
I would like to request you to use v2.0 version instead of 1.*.

V2 has better set of feature and extensibility.

**Reply by ZaheerNiazi007** [reply]
@surajair Thank you for information. This is really interested 👌
